package.json

```diff
@@ -18,28 +18,36 @@
     "node": ">=18"
   },
   "scripts": {
-    "lint": "eslint lib/*.js test/*.js index.js",
+    "lint": "eslint src/*.js test/*.js",
     "test-generate-mo": "msgfmt test/fixtures/latin13.po -o test/fixtures/latin13.mo & msgfmt test/fixtures/utf8.po -o test/fixtures/utf8.mo & msgfmt test/fixtures/obsolete.po -o test/fixtures/obsolete.mo",
     "test": "mocha",
     "preversion": "npm run lint && npm test",
-    "postversion": "git push && git push --tags"
+    "postversion": "git push && git push --tags",
+    "prepublishOnly": "npm i && tsc && npm run lint && npm run test"
   },
-  "main": "./index.js",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
```

Comment:
Why manually write the declaration and not use what is generated by `tsc`?

Reply:
You must consider that we must have continuity with past types, without changing the name. For this we will have to keep a declaration file in which we have these named 

The declaration file ["src/index.d.ts" is used by tsc](https://github.com/smhg/gettext-parser/blob/955eaa25aa0cb3e117fdf0742d66bcc6afeb3601/src/index.d.ts#L12-L37) to generate a new one that is outputted to the `lib/` directory (the build folder, BTW everything you see inside "lib" is now generated).

Anyhow the fix for the types is not yet merged at this stage in this branch you should look at #89 for that (actually is a draft)

Reply:
fixed in https://github.com/smhg/gettext-parser/commit/cbe6e8f05e6455ea79023b12af39d20077b6c1d2